beanie/odm/actions.py

```diff
@@ -128,10 +129,13 @@ async def run_actions(
             if action.__name__ in exclude:
                 continue
 
+            args = (instance, event_type)
+            count = action.__code__.co_argcount  # type: ignore
```

Comment:
This looks hacky. Let's think if it is possible to avoid it. If I'll have ideas, I'll put it here

Reply:
Python at all is a magical hacky language :)
Another option is to use `__annotation__` but it can be unset if no type hints used.

Reply:
I suppose there is a way to do it like this: `count = len(inspect.signature(action)._parameters)`